js/ui/grid_core/ui.grid_core.utils.js

```diff
@@ -10,7 +10,7 @@ var $ = require("../../core/renderer"),
     dataUtils = require("../../data/utils"),
     formatHelper = require("../../format_helper");
 
-var DATAGRID_NODATA_TEXT_CLASS = "dx-datagrid-nodata",
+var NODATA_CLASS = "nodata",
```

Comment:
What do you think about renaming to "NO_DATA_CLASS"? In the code below you use "noDataClass" for camel case notation.